mmhuman3d/core/visualization/renderer/torch3d_renderer/base_renderer.py

```diff
@@ -204,7 +204,7 @@ def init_renderer(self, cameras, lights):
                 dict(type=self.shader_type)))
         return renderer
 
-    def write_images(self, rgbs, valid_masks, images, indexs):
+    def write_images(self, rgbs, valid_masks, images, indexes):
```

Comment:
I think a more common plural form of `index` is `indices` than `indexes`.

Reply:
Yes, codespell recommends both `indices` and `indexes`. For other mm repos, it seems that they use [`indices`](https://github.com/open-mmlab/mmdetection/search?q=indices) and [`indexes`](https://github.com/open-mmlab/mmdetection/search?q=indexes) interchangable.

Reply:
As discussed offline, we may address this issue in future PR, after unifying the convention of ind/idx/index/indices/indexes.